packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -340,19 +340,73 @@ const makeDriver = async (t, initialPrice, priceBase) => {
     E(lender).getCollateralManager(aethBrand),
   ).getNotifier();
   let managerNotification = await E(managerNotifier).getUpdateSince();
-  /** @type {UserSeat<VaultKit>} */
-  let vaultSeat;
-  /** @type {VaultKit} */
-  let vault;
+
   /** @type {UserSeat<VaultKit>} */
   let lastSeat;
-  let notifier;
   let notification = {};
   let lastOfferResult;
+  const makeVaultDriver = async (collateral, debt) => {
+    /** @type {UserSeat<VaultKit>} */
+    const vaultSeat = await E(zoe).offer(
+      await E(lender).makeVaultInvitation(),
+      harden({
+        give: { Collateral: collateral },
+        want: { RUN: debt },
+      }),
+      harden({
+        Collateral: aethMint.mintPayment(collateral),
+      }),
+    );
+    const {
+      vault,
+      publicNotifiers: { vault: notifier },
+    } = await E(vaultSeat).getOfferResult();
+    t.truthy(await E(vaultSeat).hasExited());
+    return {
+      vault: () => vault,
+      vaultSeat: () => vaultSeat,
+      notification: () => notification,
+      close: async () => {
+        lastSeat = await E(zoe).offer(E(vault).makeCloseInvitation());
```

Comment:
I prefer `finalSeat`.  `lastSeat` often means *previous*.

Reply:
Ironically, this field was indeed intended to be the seat for the most recent offer, so if there were follow-up assertion tests, it would be available. Thus the next operations might be `checkPayouts(...)` which would go against the `lastSeat`

Reply:
sorry, my point is that "last" is ambiguous between "final" and "previous". Please say one or the other.

Reply:
OK I think that `currentSeat` is actually the most appropriate